changelogs/fragments/36-run-integration-on-fork-prs.yaml

```diff
@@ -0,0 +1,2 @@
+trivial:
+  - reconfigure CI integration tests to run from fork PRs, allowing external contributions to be tested
```

Comment:
Nit: the format is very particular

> - scope - description starting with a lowercase letter and ending with a period at the very end. Multiple sentences are allowed (https://github.com/reference/to/an/issue or, if there is no issue, reference to a pull request itself).

This is missing "scope", the final period, and a link to the issue or PR.

Reply:
Thank you, I appreciate it